src/main/java/mate/academy/onlinebookstore01/dto/CreateBookRequestDto.java

```diff
@@ -1,14 +1,24 @@
 package mate.academy.onlinebookstore01.dto;
 
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
 import java.math.BigDecimal;
 import lombok.Data;
 
 @Data
 public class CreateBookRequestDto {
+    @Min(1)
     private Long id;
+    @NotNull
     private String title;
+    @NotNull
     private String author;

```

Comment:
I would also validate a maximum size of text fields, to not exceed the DB column restrictions.